cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -238,6 +238,28 @@ CREATE CONSTRAINT ...
 Constraints used for string properties require an additional text index to cover the string queries properly.
 Constraints used for point properties require an additional point index to cover the spatial queries properly.
 
+a|
+label:functionality[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+SHOW INDEXES YIELD uniqueness
+----
+a|
+The `uniqueness` output has been removed altogether with the concept of index uniqueness, as it actually belongs to the constraint and not the index.
```

Comment:
```suggestion
The `uniqueness` output has been removed along with the concept of index uniqueness, as it actually belongs to the constraint and not the index.
```

In my opinion `altogether with ` should be replaced by either `along with` or `altogether, along with`

Reply:
I agree, thanks for the suggestion Phil